apis/v1alpha1/amazoncloudwatchagent_types.go

```diff
@@ -167,6 +167,9 @@ type AmazonCloudWatchAgentSpec struct {
 	// Config is the raw JSON to be used as the collector's configuration. Refer to the OpenTelemetry Collector documentation for details.
 	// +required
 	Config string `json:"config,omitempty"`
+	// Config is the raw YAML to be used as the collector's configuration. Refer to the OpenTelemetry Collector documentation for details.
+	// +optional
+	OtelConfig string `json:"otelConfig,omitempty"`
```

Comment:
Can you do a quick test with the helm chart to see if the yaml can be properly applied into the `amazoncloudwatchagent` custom resource from there?

Reply:
That's how I tested it, this field will be added to the helm chart
```
  otelConfig: 
    receivers:
      otlp:
        protocols:
          grpc:
            endpoint: "localhost:4318"

    processors:

    exporters:
      debug:

    service:
      pipelines:
        metrics:
          receivers: [ otlp ]
          exporters: [ debug ]
```

Reply:
https://github.com/aws-observability/helm-charts/pull/107

Reply:
Ah got it, looks good to me then!